invenio_deposit/views/ui.py

```diff
@@ -68,12 +76,37 @@ def index():
         return render_template(current_app.config['DEPOSIT_UI_INDEX_TEMPLATE'])
 
     @blueprint.route('/deposit/new')
+    @blueprint.route('/deposit/new/<string:deposit_type>')
```

Comment:
I would even go for GET argument: `/deposit/new?type=....` as it was proposed in https://github.com/inveniosoftware/invenio-deposit/pull/134#discussion_r82831187